commcare-support-library/src/main/java/org/commcare/commcaresupportlibrary/CommCareLauncher.java

```diff
@@ -16,8 +17,22 @@ public class CommCareLauncher {
      * @param appId Unique Id for CommCare App that CommCare should launch with
      */
     public static void launchCommCareForAppId(Context context, String appId) {
+        launchCommCareForAppId(context, appId, false);
+    }
+
+    /**
+     *
+     * @param context Android context to launch the CommCare with
+     * @param appId Unique Id for CommCare App that CommCare should launch with
+     */
+    public static void launchCommCareForAppIdFromConnect(Context context, String appId) {
```

Comment:
Support library should only host methods that are useful for external Apps to integrate with CC, Since this method is connect specific, we should not add it in support library.